lib/openstax/rescue_from/wrapped_exception.rb

```diff
@@ -0,0 +1,76 @@
+module OpenStax
+  module RescueFrom
+    class WrappedException
+      attr_reader :exception
+
+      def initialize(exception)
+        @exception = exception
+      end
+
+      def header
```

Comment:
Sure. It came directly from this bit: https://github.com/openstax/tutor-server/blob/master/config/initializers/controllers.rb#L71 -- what do you think? "title" instead, maybe?